src/header.jsx

```diff
@@ -49,6 +49,7 @@ const ViewSelector = ({ isGrid, setIsGrid, sortBy, setSortBy }) => {
     const onSelect = (ev, itemId) => {
         setIsOpen(false);
         setSortBy(itemId);
+        localStorage.setItem("sort", itemId);
```

Comment:
Since the local storage will be shared among cockpit pages it would make sense to use some more specific key? 

Something like cockpit-navigator.sort